pkgs/servers/mqtt/mosquitto/default.nix

```diff
@@ -30,13 +30,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "mosquitto";
-  version = "2.0.18";
+  version = "2.0.20";
 
   src = fetchFromGitHub {
     owner = "eclipse";
     repo = "mosquitto";
     rev = "v${version}";
-    hash = "sha256-Vs0blV2IhnlEAm0WtOartz+0vLesJfp78FNJCivRxHk=";
+    hash = "sha256-oZo6J6mxMC05jJ8RXIunOMB3kptA6FElchKlg4qmuQ8=";
   };
 
   patches = lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
Remove this patch and it builds.

Reply:
Why is that? This patch is dedicated to fix build issue on macOS, which is already fixed by upstream.

Reply:
@Aleksanaa means removing the Darwin‐only patch in `patches`, not removing the part of this diff that updates the version.